org.eclipse.jdt.launching/launching/org/eclipse/jdt/internal/launching/DetectVMInstallationsJob.java

```diff
@@ -46,6 +46,20 @@
  */
 public class DetectVMInstallationsJob extends Job {
 
+	/**
+	 * CI is a common variable defined in CI/CDI servers like Jenkins, Gitlab, Github, ... to indicate it is a CI environment
+	 */
+	private static final String ENV_CI = "CI"; //$NON-NLS-1$
```

Comment:
question: is that variable set in I-Builds?

Reply:
See initial PR description / commit message:

![grafik](https://github.com/user-attachments/assets/e93339fd-0a29-4d00-bbea-1c0f48d40ba9)

Reply:
sorry, i just still don't know if our I-Builds are common CI pipelines or too much handmade.

Reply:
If you click on the Jenkins Link it will lead you to our Jenkins and showing it is all there...

Reply:
ok, i double checked https://download.eclipse.org/eclipse/downloads/drops4/I20241215-1800/testresults/consolelogs/ep435I-unit-win32-x86_64-java17_win32.win32.x86_64_17_consolelog.txt
`[echoproperties] env.CI=true`